docs/draft/project/personas.md

```diff
@@ -0,0 +1,110 @@
+OLM Personas
+
+# About this doc
```

Comment:
```suggestion
```

The overview info, what is the doc about is added in all docs after the title. So, I do not think that we need the sub-section title